### ISSUE TITLE
Avoid markdown etc

### ISSUE THREAD

**Issue by svenseeberg** [digitalfabrik/integreat-chat#88]
Sometimes answers by the LLM are styled in Markdown. However, we only print plain text back to the user. We should therefore try to avoid Mardown etc. Maybe we can add this to the prompt?

**Comment by dasgoutam** [comment]
Do we have an example query where this occurs,  or is it random?

**Comment by svenseeberg** [comment]
Yes, the question `Are there any specific regulations for Ukrainians in Germany?` did yield such a result.
